src/clients/firstboot.rb

```diff
@@ -61,6 +62,8 @@ def main
       # firstboot workflow expect them to be initialized (bug #335979)
       PackageCallbacksInit.InitPackageCallbacks
 
+      UI.SetProductName(Product.name)
```

Comment:
is this UI call already in SLE15-GA? Do you test it?

Reply:
:thinking: :thinking: :thinking: 

This is a change actually introduced in `master` which was backported after the previous PR in `SLE-12-SP4`.

I did tests (manually), but not with this code base.

Give me a few minutes. I'll try to test it.

Reply:
Good news. It works!<sup>1</sup>

<details>
<summary>Click to see the screenshots</summary>

![Screenshot_opensusetumbleweed_2019-05-13_14:17:27](https://user-images.githubusercontent.com/1691872/57624573-264c9300-758a-11e9-9cc6-afd52c7a69a4.png)

![Screenshot_opensusetumbleweed_2019-05-13_14:19:13](https://user-images.githubusercontent.com/1691872/57624581-29478380-758a-11e9-98de-288d92ea5436.png)
</details>

---

 <sub>after the fix made in #69</sub>